TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,27 @@
 
 ## Unreleased
 
+### Add abtest package and tracks events to validate #6536
```

Comment:
This is something we've only just started talking about, but this kind of testing instruction isn't really required in this file, as it will be used to pass to our testers and this isn't testable for anyone but a developer